server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java

```diff
@@ -48,8 +52,10 @@ public static void printJsonAsTable(String jsonString) {
             }
 
         } catch (JsonMappingException e) {
+            LOGGER.error("printJsonAsTable happen JsonMappingException,jsonString:{}",jsonString,e);
```

Comment:
can we please format this better and add a meaningful message?

Reply:
Ok，it's no problem，thanks